validators/index.js

```diff
@@ -3,6 +3,14 @@
 const semver = require('semver');
 const npmPkg = require('validate-npm-package-name');
 
+const server = value => {
+    if (new RegExp('^https?://[a-zA-Z0-9-_./]+(:[0-9]+)?').test(value)) {
+        return value.toLowerCase();
+    }
+    throw new Error('Parameter "server" is not valid');
+};
+module.exports.server = server;
```

Comment:
Could we call this `origin`?  In a [WHATWG URL scheme](https://nodejs.org/api/url.html#url_url_strings_and_url_objects) this part is called `origin`.

Reply:
sure thing

Reply:
Aside, and classic problem with writing shared code. When we use the origin validator to validate an input to the cli called "server", the error message is going to be confusing: "'Parameter "origin" is not valid'" Should it be `origin` in the CLI as well?

Reply:
I think these errors should contain an message which give sense in error logs. Error messages we expose to the end user should be crafted for each use case. 

Ex; In core the validators do throw, but I catch these errors and send different errors to the http framework depending on the need. 

Ex: the `pkg put` route yeld a [400 error](https://github.com/asset-pipe/core/blob/master/lib/handlers/pkg.put.js#L182-L188) when a validator is not passed while the `pkg get` route yeld a [404 error](https://github.com/asset-pipe/core/blob/master/lib/handlers/pkg.get.js#L9-L16).